src/posts/posts.service.ts

```diff
@@ -118,7 +127,8 @@ export class PostsService {
                 updated: new Date()
             }
         })
-        return updatedpost;
+        const action = await this.actionsService.addAction('Update',userId,'Post',post.id,updatedpost);
```

Comment:
here you have hadcoded strings. Store them in variables and reuse? insted of hardcode them